data/movies.json

```diff
@@ -1 +1 @@
-[{"id":842,"name":"test","silent":false,"publish_date":"2022-01-01"},{"id":850,"name":"testing","silent":true,"publish_date":"2022-05-03"},{"id":212,"name":"New Movie","silent":false,"publish_date":"2019-01-01"}]
\ No newline at end of file
+[{"id":627,"name":null,"silent":null,"publish_date":null},{"id":1524,"name":null,"silent":null,"publish_date":null}]
```

Comment:
### Don't change files related to other modules. Because our app is following `SOLID` principles so it won't be needed.
-  This is corrupting the data.
- If needs to be changed please talk with the responsible developer on the module first, this is required in a group work.

Reply:
Okay @RitobrotoMukherjee